TROUBLESHOOTING.md

```diff
@@ -193,7 +193,9 @@ only be enabled for debugging.
   moving to the next step.
 
 * ```XLA_USE_BF16```: If set to 1, tranforms all the _PyTorch_ _Float_ values into _BiFloat16_
-  when sending to the _TPU_ device.
+  when sending to the _TPU_ device. Note that when using `XLA_USE_BF16=1` tensor arithmetic will
+  be done in reduced precision and so tensors such as loss values and other metrics will not be
+  useful. To get accurate loss and other metric values, user manual mixed precision.
```

Comment:
The problem w/ loss + bf16 arises when user accumulates tensors, the numerator stops growing due to lack of precision but denom. keeps growing. so if one looks at the raw loss tensors at a given step, in bf16, that is definitely a meaningful value. but if one looks at the **average** loss over 100 steps, computed as `sum(losses) / 100`, than that's not meaningful.

Reply:
Updated comments PTAL.